core/lib/server/pub.ts

```diff
@@ -513,6 +514,9 @@ export const createPubRecursiveNew = async <Body extends CreatePubRequestBodyWit
 						.returningAll()
 				).execute()
 			: [];
+
+		const pub = await getPlainPub(newPub.id, trx).executeTakeFirstOrThrow();
```

Comment:
so i added this bc i think it's nice to, in api responses etc, return an accurate `title` for the recently created pub.

we need to do it somewhat like this, bc we cannot accurately reconstruct what the title would be from the input alone (unless we fetched the pubtype X pubfields for the pub, which I tihnk is slower than just fetching the pub). this does make this function slightly slower, but i don't think that's that big of a deal.